Swiftfin/Components/CircularProgressView.swift

```diff
@@ -9,6 +9,7 @@
 import Defaults
 import SwiftUI
 
+// TODO: Can we remove this in iOS 16?
 // SwiftUI gauge style not available on iOS 15
```

Comment:
I made this comment confusing. I would like to possible replace the underlying implementation of this type with a `Gauge` with the `.gaugeStyle(.accessoryCircularCapacity)` applied. Just change these comments for a `TODO` with that.